specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9093 @@
+            "$ref": "./examples/IntegrationRuntimes_DisableInteractiveQuery.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}/enableInteractiveQuery": {
+      "post": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Enables interactive querying",
+        "operationId": "IntegrationRuntimes_EnableInteractiveQuery",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/EnableInteractiveQueryForIntegrationRuntimeRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OkResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_EnableInteractiveQuery": {
+            "$ref": "./examples/IntegrationRuntimes_EnableInteractiveQuery.json"
+          }
+        }
+      }
+    },
+    "/managedvirtualnetworks/{managedVNetName}": {
+      "get": {
+        "tags": [
+          "ManagedVirtualNetworks"
+        ],
+        "description": "Gets a ManagedVirtualNetwork",
+        "operationId": "ManagedVirtualNetworks_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "managedVNetName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ManagedVirtualNetwork"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedVirtualNetworks_Get": {
+            "$ref": "./examples/ManagedVirtualNetworks_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedVirtualNetworks"
+        ],
+        "description": "Creates or Updates a ManagedVirtualNetwork",
```

Comment:
Today creating multiple managed vnet is not be allowed? The API implementation allows to create vnets other than 'default'

Reply:
Since this doesn't change the API signature itself and is limited to service side change, I'm resolving this conversation here and make the change in Scanning service. Please feel free to reopen for any other questions/concerns.

Reply:
Changed this to enum